backend/src/emissions/schema.py

```diff
@@ -850,7 +855,49 @@ def mutate(root, info, input=None):
             return SetWorkingGroup(user=user, success=success)
         except ValidationError as e:
             return SetWorkingGroup(user=user, success=success, errors=e)
+        
 
+class RemoveUserFromWorkingGroup(graphene.Mutation):
+    """GraphQL mutation to remove a user from a working group"""
+    
+    class Arguments:
+        """Input structure defined by input type"""
+        
+        input = RemoveUserFromWorkingGroupInput()
+        
+    success = graphene.Boolean()
+    
+    @staticmethod
+    @login_required
+    @representative_required
+    def mutate(root, info, input):
+        """Process incoming data"""
+        user = info.context.user
+        
+        user_subset = CustomUser.objects.filter(id=input.user_id)
+        
+        if len(user_subset) == 0:
+            raise GraphQLError(
+                "The user you were trying to remove was not found. Please contact your administrator."
+            )
+        else:
+            user_to_remove = user_subset[0]
+        
+        if user_to_remove.is_representative is True:
```

Comment:
Should we also check that the user is member of the representative's working group to prevent representatives from removing users from other groups? Or is this not possible anyways?

Reply:
This is just prevented by the UI... I will see if I can implement this :)